shared/src/main/scala/amf/core/client/scala/AMFResult.scala

```diff
@@ -13,4 +14,9 @@ import amf.core.client.scala.validation.{AMFValidationResult, ReportConformance}
   */
 case class AMFResult(baseUnit: BaseUnit, results: Seq[AMFValidationResult]) extends AMFObjectResult(baseUnit, results)
 
+class AMFParseResult(override val baseUnit: BaseUnit, override val results: Seq[AMFValidationResult])
+    extends AMFResult(baseUnit, results) {
+  def rootSpec: Spec = baseUnit.sourceSpec.getOrElse(Amf)
```

Comment:
Will we expose rootSpec in AMFResult and also in BaseUnit?

Reply:
I will leave it like that for the moment.